MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -4927,19 +4927,14 @@ public TargetRoll getTargetForMaintenance(IPartWork partWork, Person tech) {
             // "astech days" used over
             // the cycle and take the average per day rounding down as our team
             // size
-            int helpMod = 0;
-            if (null != partWork.getUnit() && partWork.getUnit().isSelfCrewed()) {
-                int hits = 0;
-                if (null != partWork.getUnit().getEntity().getCrew()) {
-                    hits = partWork.getUnit().getEntity().getCrew().getHits();
-                } else {
-                    hits = 6;
-                }
-                helpMod = getShorthandedModForCrews(hits);
+            final int helpMod;
+            if (partWork.getUnit().isSelfCrewed()) {
+                helpMod = getShorthandedModForCrews((partWork.getUnit().getEntity().getCrew() == null)
```

Comment:
I have an extremely hard time parsing this expression. Are you passing the results of a ternary operator into a function and then assigning the resulting value to helpMod?

This really needs to be put back to the way it was or have a comment explaining exactly what it does.

Reply:
Moved the logic into Campaign::getShorthandedModForCrews, as it was being duplicated and just makes more sense based on the name to be calculated there.